ci-operator/config/openshift/multiarch/openshift-multiarch-master__nightly-4.11.yaml

```diff
@@ -136,7 +136,11 @@ tests:
     dependencies:
       OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:multi-latest
     env:
+      ADDITIONAL_WORKER: "2"
       BASE_DOMAIN: aws-2.ci.openshift.org
+      HETEROGENEOUS: "true"
+      WORKER_ARCHITECTURE: aarch64
```

Comment:
Hi @deepsm007 , can we rename WORKER_ARCHITECTURE to `ADDITIONAL_WORKERS_ARCHITECTURE` and the same for `ADDITIONAL_WORKER_VM_TYPE`? It would make clear that these values only affect the additional/patched workers, not the "normal" ones.